package_manager/parse_metadata_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import os
+
+from parse_metadata import parse_package_metadata
+
+class TestParseMetadata(unittest.TestCase):
+
+    def setUp(self):
+        dir = os.path.dirname(__file__)
```

Comment:
nit: dir is technically a builtin you're shadowing here. A different name would be better.